_benchmark/commands/command-flags.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Command flags
+nav_order: 51
+parent: Command reference
+---
+
+OpenSearch Benchmark (OSB) uses command line flags to change OpenSearch Benchmark's behavior. Not all flags can be uses with each command. To find out which flags are supported by a specific command, enter `opensearch-benchmark <command> --h`.
+
+All command flags are added to a command using the following syntax:
+
+```bash
+opensearch-benchmark <command> --<command-flag>
+```
+
+Flags that accept comma-seperated values such `--telemetry` can also accept a JSON array. This can be defined by passing in a file path ending in `.json` or inline as a JSON string.
+
+- Comma-seperated values: `--test-procedure="4gheap,trial-license"`
+- JSON file: `cat test-procedure.json`
```

Comment:
`cat` can be removed because we're pointing out the file extension.

Reply:
On second thought, let's switch this with a command line since this is what the other two examples are doing:
```
--target-hosts="opensearch-target-host.json"
```